canopycover.py

```diff
@@ -0,0 +1,358 @@
+#!/usr/bin/env python3
+"""Calculates canopy coverage for plots in georeferenced images
+"""
+
+import argparse
+import copy
+import json
+import logging
+import os
+from typing import Union
+import numpy as np
+import dateutil.parser
+from agpypeline import entrypoint, algorithm, geoimage
+from agpypeline.environment import Environment
+from osgeo import gdal, ogr
+
+from configuration import ConfigurationCanopycover
+
+
+# Array of trait names that should have array values associated with them
+TRAIT_NAME_ARRAY_VALUE = ['canopy_cover', 'site']
```

Comment:
why is site a trait name?

Reply:
naming convention from TERRA REF. It can be changed to something clearer